unity-client/Assets/Scripts/MainScripts/DCL/Components/Audio/Tests/AudioTests.cs

```diff
@@ -220,6 +221,171 @@ public IEnumerator AudioClipAttachedGetsReplacedOnNewAttachment()
                 scene, CLASS_ID.AUDIO_CLIP);
         }
 
+        [UnityTest]
+        public IEnumerator VolumeWhenAudioCreatedWithNoUserInScene()
+        {
+            DCLCharacterController.i.Teleport(JsonConvert.SerializeObject(new
+            {
+                x = -1f,
+                y = DCLCharacterController.i.transform.position.y,
+                z = -1f
+            }));
+            yield return null;
+
+            yield return LoadAudioClip(scene,
+                audioClipId: "audioClipTest",
+                url: DCL.Helpers.Utils.GetTestsAssetsPath() + "/Audio/Train.wav",
+                loop: true,
+                loading: true,
+                volume: 1f,
+                waitForLoading: true);
+
+            DecentralandEntity entity = TestHelpers.CreateSceneEntity(scene);
+            yield return null;
+
+            yield return CreateAudioSource(scene,
+                entityId: entity.entityId,
+                audioClipId: "audioClipTest",
+                playing: true);
+
+            DCLAudioSource dclAudioSource = entity.gameObject.GetComponentInChildren<DCLAudioSource>();
+            yield return dclAudioSource.routine;
+
+            AudioSource unityAudioSource = dclAudioSource.GetComponentInChildren<AudioSource>();
+
+            // Check the volume
+            Assert.AreEqual(unityAudioSource.volume, 0f);
+        }
+
+        [UnityTest]
+        public IEnumerator VolumeWhenAudioCreatedWithUserInScene()
+        {
+            DCLCharacterController.i.Teleport(JsonConvert.SerializeObject(new
+            {
+                x = 8f,
+                y = DCLCharacterController.i.transform.position.y,
+                z = 8f
```

Comment:
As propossed in the tests below, cannot we just change the scriptable objets the Audio component is reacting to?

Reply:
done